catalog/app/containers/Bucket/PackageUpdateDialog.tsx

```diff
@@ -59,6 +60,16 @@ interface Uploads {
   }
 }
 
+interface LocalEntry {
+  path: string
+  file: Exclude<PD.FilesState['added'][string], S3File>
```

Comment:
Probably, it's better to go from the opposite direction. Define `LocalFile` or `AddiedFile` in contrast to `S3File`, and define `FilesState` using this `LocalFile` type rather then define `LocalEntry` using `FilesState`

Reply:
yeah, makes sense